my_game.py

```diff
@@ -102,27 +84,46 @@ def __init__(self, center_x, center_y, lives, scale):
         self.change_y = self.speed * math.cos(self.angle)
         self.center_x = center_x
         self.center_y = center_y
+        self.is_imortal = is_imortal
+        self.die_cooldown = 30
+        self.alpha = 255
+        self.respawning = False
+
+    def reset(self):
+        """
+        What happens to the player when it loses a life.
+        """
+        # It 100% transparent
+        self.respawning = True
+        self.alpha = 0
+        self.die_cooldown = 90
 
     def update(self):
```

Comment:
Let's change this to _on_update(self, delta_time)_. Then use the _delta_time_ value to decrement your _die_cooldown_.